cli/cmd/plugin/ui/web/tanzu-ui/src/views/welcome/RolloverBanner/RolloverBanner.tsx

```diff
@@ -32,8 +32,8 @@ const RolloverBanner = () => {
         {
             logo: TceLogo,
             icon: '',
-            backgroundImage: TceExperienceBg,
-            title: 'Community-supported experience',
+            // backgroundImage: TceExperienceBg,
```

Comment:
Hi Shimon, was it intentional to remove these background images?

Reply:
I will reenstate them. These were the files that weren't check in before. Good catch